src/routes/activity/repository.js

```diff
@@ -0,0 +1,25 @@
+const { db, errors } = require('../../shared')
+
+const { InsertActivityError } = errors.system
+
+const insertActivity = async (activity) => {
+    const { idAlternative, user_id, device_id, device_serial, idQuestion,  } = activity
+    const result = await db.mysql.query(
+        'CALL `insert_activity`(?, ?, ?, ?, ?)',
+        [
+            idAlternative,
+            user_id,
+            device_id,
+            device_serial,
+            idQuestion
+        ]
+    )
+
+    if (!Array.isArray(result) && !result.affectedRows)
+        throw new InsertActivityError()
+
+    console.log(result)
+    return result
```

Comment:
Result is a array you might want to return the last query executed on the procedure
```suggestion
    return result?.at(0)?.at(0)
```